code/game/objects/structures/crates_lockers/closets/secure/engineering.dm

```diff
@@ -1,149 +1,150 @@
-/obj/structure/closet/secure_closet/engineering_chief

```

Comment:
What went wrong; this concerns me.

Reply:
Oh, looks like they used the website to resolve conflicts and it screwed with the format

Reply:
@MrMagolor pleased fix this.